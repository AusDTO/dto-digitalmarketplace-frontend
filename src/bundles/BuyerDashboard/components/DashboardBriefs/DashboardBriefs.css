.summary-item-field a {
    white-space: nowrap
}

.summary-item-field-heading-first {
    font-size: 1rem
}

.create-opportunity-link {
    padding-bottom: 0px
}

.create-opportunity-link p {
    margin-bottom: 1rem
}

section.live-brief-container {
    margin-top: 3.5rem
}

.dashboard-main {
    width: 74.48861%
}

@media (min-width: 768px) {
    tbody tr td h5 {
        display: none
    }
}

.item-field-first-full {
    width: 25%
}

.item-field-first {
    width: 25%
}

.item-field-second-left {
    width: 25%;
    padding-top: 0px
}

.item-field-second-right {
    width: 25%
}

.item-field-third-left {
    width: 25%;
    padding-top: 0px
}

.item-field-third-right {
    width: 25%;
    padding-top: 0px;
    text-align: right
}

.item-field-third-right.teamview {
    text-align: left
}

@media (max-width: 767px) {
    .item-field-third-right.teamview {
        text-align: right
}
}

.item-field-fourth {
    width: 25%;
    padding-top: 0px;
    text-align: right
}

@media (max-width: 767px) {
    .dashboard-main {
        width: 100%
    }

    section h2 {
        border-bottom: 3px solid #666666;
        padding-bottom: 15px;
        margin-bottom: 0px
    }

    .summary-item-field-heading {
        display: none
    }


    .table-row {
        position: absolute;
        top: -9999px;
        left: -9999px
    }

    .closed-item-row {
        display: block
    }

    .closed-item-field-title {
        display: inline-block;
        width: 33.3% !important;
        border: hidden !important
    }

    .closed-item-field-date {
        display: inline-block;
        width: 33.3% !important;
        border: hidden !important
    }

    .closed-item-field-actions {
        display: inline-block;
        width: 33.3% !important
    }

    .closed-item-field-actions a {
        white-space: nowrap
    }

    .item-field {
        display: inline-block;
        width: 33.3% !important;
        border: hidden !important
    }

    .item-field-first-full {
        display: inline-block;
        width: 100%;
        padding-left: 0px
    }

    .item-field-first {
        display: inline-block;
        width: 50%;
        padding-left: 0px
    }

    .item-field-second-left {
        display: inline-block;
        width: 50%;
        text-align: left;
        padding-left: 0px
    }

    .item-field-second-right {
        display: inline-block;
        width: 50%;
        text-align: right
    }

    .item-field-third-left {
        display: inline-block;
        width: 50%;
        text-align: left
    }

    .item-field-third-right {
        display: inline-block;
        width: 50%;
        text-align: right;
    }

    .item-field-fourth {
        display: inline-block;
        width: 50%;
        text-align: right
    }

    .item-field-fourth-empty {
        display: none
    }

    .item-field a {
        overflow-wrap: break-word;
    }

    table.brief-table tr td {
        padding: 1em 1.2em 1em 0.4em
    }

    table.brief-table tr td.item-field-second-left {
        padding: 0 1.2em 1em 0.4em
    }
    
    table.brief-table tr td.item-field-third-left {
        padding: 0 1.2em 1em 0.4em
    }
    
    table.brief-table tr td.item-field-third-right {
        padding: 0 1.2em 1em 0.4em
    }

     table.brief-table tr td.item-field-fourth {
        padding: 0 1.2em 1em 0.4em
    }

    table.brief-table thead tr th.center-header {
        padding-right: 20px;
        text-align: left
    }

    table.brief-table thead tr th.right-header {
        text-align: right;
        width: 10%
    }

    tbody tr td h5 {
        margin: 0px
    }
}

table.brief-table {
    margin: 1em 0 1em
}

table.brief-table caption {
    font-size: 1.0625rem;
    padding-bottom: 1em;
    text-align: left
}

table.brief-table td {
    padding: 1em 1.2em 1em 0.4em
}

table.brief-table tr {
    border-bottom: 1px solid #bebebe
}

table.brief-table tr:first-child {
    border-top: 1px solid #bebebe
}

table.brief-table thead tr th {
    font-weight: 700;
    line-height: 18px
}

table.brief-table tr:nth-child(even) {
    background-color: white

}

.left-header {
    text-align: left;
    padding-left: 5px
}

.center-header {
    text-align: left
}

.right-header {
    color: #313131;
    text-align: right;
    padding-right: 15px
}

table.brief-table tr td, table.brief-table tr th {
    font-size: .875rem;
    font-weight: 400;
    vertical-align: top;
}

table.brief-table p, table.brief-table .sidebar h2 span, .sidebar h2 table.brief-table span, table.brief-table .sidebar .display span, .sidebar .display table.brief-table span {
    font-size: .875rem
}

table.brief-table ul, table.brief-table ol {
    font-size: .875rem;
    padding-left: 1.5em
}

table.brief-table.summary-item-body {
    width: 100%;
    margin-top: 0px
}

table.brief-table.summary-item-body tr:first-child {
    border-top: 0
}

table.brief-table.summary-item-body thead tr:first-child {
    border-bottom: 3px solid #666666
}

table.brief-table.summary-item-body-no-bottom-border-row-bold-borders-text-large tr:first-child {
    border-top: 0
}

table.brief-table .summary-item-field-headings th {
    border-bottom: 0
}

table.brief-table .summary-item-free-text span {
    white-space: pre-wrap
}

tbody tr td ul {
    list-style: none
}

