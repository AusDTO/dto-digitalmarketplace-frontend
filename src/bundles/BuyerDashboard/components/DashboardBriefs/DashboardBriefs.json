{
  "team": {
    "view": "mybriefs",
    "currentUserName": "Matt Smith",
    "teamName": "Digital Transformation Agency",
    "briefs": {
      "live": [
        {
          "clarificationQuestions": [],
          "createdAt": "2016-10-06T01:14:30.360862+00:00",
          "dates": {
            "answers_close": null,
            "application_open_weeks": "1 week",
            "closing_date": null,
            "closing_time": null,
            "hypothetical": {
              "answers_close": "2017-05-05T08:00:00+00:00",
              "application_open_weeks": "1 week",
              "closing_date": "2017-05-08",
              "closing_time": "2017-05-08T08:00:00+00:00",
              "published_date": "2017-05-01",
              "questions_close": "2017-05-03T08:00:00+00:00"
            },
            "published_date": null,
            "questions_close": null
          },
          "frameworkFramework": "dsp",
          "frameworkName": "Digital Service Professionals",
          "frameworkSlug": "digital-service-professionals",
          "frameworkStatus": "live",
          "id": 119,
          "links": {
            "framework": "http://localhost:5000/frameworks/digital-service-professionals",
            "self": "http://localhost:5000/briefs/119"
          },
          "lot": "digital-outcome",
          "lotName": "Digital outcome",
          "lotSlug": "digital-outcome",
          "requirementsLength": "1 week",
          "status": "live",
          "title": "Matts Brief",
          "unanswered_optional": 7,
          "unanswered_required": 17,
          "updatedAt": "2016-10-06T03:11:46.824672+00:00"
        }
      ],
      "closed": [
        {
          "clarificationQuestions": [],
          "createdAt": "2016-10-06T01:14:30.360862+00:00",
          "dates": {
            "answers_close": null,
            "application_open_weeks": "1 week",
            "closing_date": null,
            "closing_time": null,
            "hypothetical": {
              "answers_close": "2017-05-05T08:00:00+00:00",
              "application_open_weeks": "1 week",
              "closing_date": "2017-05-08",
              "closing_time": "2017-05-08T08:00:00+00:00",
              "published_date": "2017-05-01",
              "questions_close": "2017-05-03T08:00:00+00:00"
            },
            "published_date": null,
            "questions_close": null
          },
          "frameworkFramework": "dsp",
          "frameworkName": "Digital Service Professionals",
          "frameworkSlug": "digital-service-professionals",
          "frameworkStatus": "live",
          "id": 119,
          "links": {
            "framework": "http://localhost:5000/frameworks/digital-service-professionals",
            "self": "http://localhost:5000/briefs/119"
          },
          "lot": "digital-outcome",
          "lotName": "Digital outcome",
          "lotSlug": "digital-outcome",
          "requirementsLength": "1 week",
          "status": "closed",
          "title": "Matts Brief",
          "unanswered_optional": 7,
          "unanswered_required": 17,
          "updatedAt": "2016-10-06T03:11:46.824672+00:00"
        }
      ],
      "draft": [
        {
          "clarificationQuestions": [],
          "createdAt": "2016-10-06T01:14:30.360862+00:00",
          "dates": {
            "answers_close": null,
            "application_open_weeks": "1 week",
            "closing_date": null,
            "closing_time": null,
            "hypothetical": {
              "answers_close": "2017-05-05T08:00:00+00:00",
              "application_open_weeks": "1 week",
              "closing_date": "2017-05-08",
              "closing_time": "2017-05-08T08:00:00+00:00",
              "published_date": "2017-05-01",
              "questions_close": "2017-05-03T08:00:00+00:00"
            },
            "published_date": null,
            "questions_close": null
          },
          "frameworkFramework": "dsp",
          "frameworkName": "Digital Service Professionals",
          "frameworkSlug": "digital-service-professionals",
          "frameworkStatus": "live",
          "id": 119,
          "links": {
            "framework": "http://localhost:5000/frameworks/digital-service-professionals",
            "self": "http://localhost:5000/briefs/119"
          },
          "lot": "digital-outcome",
          "lotName": "Digital outcome",
          "lotSlug": "digital-outcome",
          "requirementsLength": "1 week",
          "status": "draft",
          "title": "Matts Brief",
          "unanswered_optional": 7,
          "unanswered_required": 17,
          "updatedAt": "2016-10-06T03:11:46.824672+00:00"
        }
      ]
    },
    "members": [
      {
      "email_address": "marketplace@digital.gov.au",
      "email_domain": "digital.gov.au",
      "id": 452,
      "name": "AJ"
    },
    {
      "email_address": "alex.sadleir+s@digital.gov.au",
      "email_domain": "digital.gov.au",
      "id": 1921,
      "name": "Alex"
    }
    ]
  }
}
