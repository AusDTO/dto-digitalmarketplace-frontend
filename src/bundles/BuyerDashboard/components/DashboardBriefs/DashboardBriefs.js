import React from 'react';
import isEmpty from 'lodash/isEmpty';
import format from 'date-fns/format';
import parse from 'date-fns/parse'
import {sortByDate} from '../../../../helpers'

import './DashboardBriefs.css'

export const DashboardBriefs = props => {
  let {
    live = [],
    closed = [],
    draft = []
  } = props.briefs;
  let teamBriefView = props.teamBriefView;

  // sortByDate(dateArray, briefTypeString, sortDirections)
  live = Array.from(sortByDate(live, 'live', 'asc')) || live;
  closed = Array.from(sortByDate(closed, 'closed', 'asc')) || closed;
  draft = Array.from(sortByDate(draft, 'draft', 'asc')) || draft;

  return (
    <div styleName="dashboard-main">
      <DraftBriefs draft={draft} teamBriefView={teamBriefView}/>

      <LiveBriefs live={live} teamBriefView={teamBriefView}/>

      <ClosedBriefs closed={closed} teamBriefView={teamBriefView}/>

      <div>
        <h3>Need a hand?</h3>
        <span>
          <a href="/buyers-guide">Read the buyers guide </a>
          or <a href={"mailto:marketplace@digital.gov.au"}>
            send us an email </a>
          - We can help you write your brief.
        </span>
      </div>
    </div>
  )
};

DashboardBriefs.propTypes = {
  live: React.PropTypes.array,
  closed: React.PropTypes.array,
  draft: React.PropTypes.array,
  teamBriefView: React.PropTypes.bool
};

export const DraftBriefs = props => {
  const {draft, teamBriefView} = props

  return (
    <section>
      {(teamBriefView
        ? <h2>Team drafts</h2>
        : <h2>Drafts</h2>)}
      {(isEmpty(draft)
        ? "You haven't started any opportunities."
        : <table styleName="brief-table summary-item-body">
          {(teamBriefView
            ? <thead styleName="summary-item-field-heading">
                <tr>
                  <th styleName="left-header">
                    Brief Preview</th>
                  <th styleName="center-header">Author</th>
                  <th styleName="center-header">Created</th>
                  <th styleName="right-header"></th>
                </tr>
              </thead>
            : <thead styleName="summary-item-field-heading">
              <tr>
                <th styleName="left-header">Brief Overview</th>
                <th styleName="center-header">Created</th>
                <th styleName="right-header">Questions left</th>
              </tr>
            </thead>)}

          {draft.map((item, i) => {
            return (
              <tbody key={i}>
                {(teamBriefView && <tr className="summary-item-row">
                  <td styleName="item-field-first-full">
                    <h5>Brief overview</h5>
                    <a href={"/" + item.frameworkSlug + "/opportunities/" + item.id}>
                      {item.title}
                    </a>
                  </td>
                  <td styleName="item-field-second-left">
                    <h5>Author</h5>
                    {item.author}
                  </td>
                  <td styleName="item-field-third-right teamview">
                    <h5>Created</h5>
                    {formatDate(item.createdAt)}
                  </td>
                  <td styleName="item-field-fourth-empty"></td>
                </tr>)}
                {(!teamBriefView && <tr className="summary-item-row">
                  <td styleName="item-field-first-full">
                    <h5>Brief overview</h5>
                    <a
                      href={"/buyers/frameworks/".concat(item.frameworkSlug, "/requirements/", item.lotSlug, "/", item.id)}>
                      {item.title}
                    </a>
                  </td>
                  <td styleName="item-field-second-left">
                    <h5>Created</h5>
                    {formatDate(item.createdAt)}
                  </td>
                  <td styleName="item-field-third-right">
                    <h5>Questions left</h5>
                    <div>{item.unanswered_required} required</div>
                    <div>{item.unanswered_optional} optional</div>
                  </td>
                </tr>)}
              </tbody>
            )
          })}
        </table>)}
    </section>
  )
};

DraftBriefs.propTypes = {
  draft: React.PropTypes.array.isRequired
};

export const ClosedBriefs = props => {
  const {closed, teamBriefView} = props;

  return (
    <section>
      {(teamBriefView
        ? <h2>Closed team briefs</h2>
        : <h2>Closed briefs</h2>)}
      {(isEmpty(closed)
        ? "You don't have any closed briefs."
        : <table styleName="brief-table summary-item-body">
          <thead styleName="summary-item-field-heading">
            {(teamBriefView
              ? <tr>
                  <th styleName="left-header">Brief Preview</th>
                  <th styleName="center-header">Author</th>
                  <th styleName="center-header">Closed</th>
                  <th styleName="right-header"></th>
                </tr>
              : <tr>
                <th styleName="left-header">Brief Overview</th>
                <th styleName="center-header">Closed</th>
                <th styleName="right-header">Work Order</th>
              </tr>)}
          </thead>

          {closed.map((item, i) => {
            return (
              <tbody key={i}>
                {(teamBriefView && <tr className="summary-item-row">
                  <td styleName="item-field-first-full">
                    <h5>Brief preview</h5>
                    <a href={"/" + item.frameworkSlug + "/opportunities/" + item.id}>
                      {item.title}
                    </a>
                  </td>

                  <td styleName="item-field-second-left">
                    <h5>Author</h5>
                    {item.author}
                  </td>
                  <td styleName="item-field-third-right teamview">
                    <h5>Closed</h5>
                    {formatDate(item.dates.closing_date)}
                  </td>
                  <td styleName="item-field-fourth-empty"></td>
                </tr>)}

                {(!teamBriefView && <tr className="summary-item-row">
                  <td styleName="item-field-first-full">
                    <h5>Brief preview</h5>
                    <a
                      href={"/buyers/frameworks/".concat(item.frameworkSlug, "/requirements/", item.lotSlug, "/", item.id)}>
                      {item.title}
                    </a>
                  </td>

                  <td styleName="item-field-second-left">
                    <h5>Closed</h5>
                    {formatDate(item.dates.closing_date)}
                  </td>
                  <td styleName="item-field-third-right">
                    <h5>Work order</h5>
                    <span style={{ marginRight: '5px'}}>
                      <a href={`/buyers/frameworks/${item.frameworkSlug}/requirements/${item.lotSlug}/${item.id}/responses`}> 
                        View responses </a>
                    </span>
                    <span>
                    {(!item.work_order_id
                      ? <a
                          href={"/buyers/frameworks/".concat(item.frameworkSlug, "/requirements/", item.lotSlug, "/", item.id, '/work-orders/create')}>
                          Create work order
                        </a>
                      : <a href={'/work-orders/' + item.work_order_id}>
                        Edit work order
                      </a>)}
                      </span>
                  </td>
                </tr>)}
              </tbody>
            )
          })}

        </table>)}
    </section>
  )
};

ClosedBriefs.propTypes = {
  closed: React.PropTypes.array.isRequired
};

export const LiveBriefs = props => {
  const {live, teamBriefView} = props;

  return (
    <section styleName="live-brief-container">
      {(teamBriefView
        ? <h2>Live team briefs</h2>
        : <h2>Live briefs</h2>)}
      {(isEmpty(live)
        ? "You don't have any live briefs."
        : <table styleName="brief-table summary-item-body">

          {(teamBriefView
            ? <thead styleName="summary-item-field-heading">
                <tr>
                  <th styleName="left-header">Brief Overview</th>
                  <th styleName="center-header">Author</th>
                  <th styleName="center-header">Closing</th>
                  <th styleName="right-header">Q&A</th>
                </tr>
              </thead>
            : <thead styleName="summary-item-field-heading">
              <tr>
                <th styleName="left-header">Brief Overview</th>
                <th styleName="center-header">Closing</th>
                <th styleName="right-header">Q&A</th>
              </tr>
            </thead>)}

          {live.map((item, i) => {
            return (
              <tbody key={i}>
                {(teamBriefView && <tr className="summary-item-row">
                  <td styleName="item-field-first">
                    <h5>Brief preview</h5>
                    <a
                      href={"/buyers/frameworks/".concat(item.frameworkSlug, "/requirements/", item.lotSlug, "/", item.id)}>
                      {item.title}
                    </a>
                  </td>
                  <td styleName="item-field-second-right">
                    <h5>Author</h5>
                    {item.author}
                  </td>
                  <td styleName="item-field-third-left">
                    <h5>Closing</h5>
                    {formatDate(item.dates.closing_date)}
                  </td>
                  <td styleName="item-field-fourth">
                    <h5>Q&A</h5>
                    <a
                      href={"/buyers/frameworks/digital-marketplace/requirements/".concat(item.lotSlug, "/", item.id, "/supplier-questions")}>
                      Answer a question
                    </a>
                  </td>
                </tr>)}
                {(!teamBriefView && <tr className="summary-item-row" key={i}>
                  <td styleName="item-field-first-full">
                    <h5>Brief preview</h5>
                    <a
                      href={"/buyers/frameworks/".concat(item.frameworkSlug, "/requirements/", item.lotSlug, "/", item.id)}>
                      {item.title}
                    </a>
                  </td>
                  <td styleName="item-field-second-left">
                    <h5>Closing</h5>
                    {formatDate(item.dates.closing_date)}
                  </td>
                  <td styleName="item-field-third-right">
                    <h5>Q&A</h5>
                    <a
                      href={"/buyers/frameworks/digital-marketplace/requirements/".concat(item.lotSlug, "/", item.id, "/supplier-questions")}>
                      Answer a question
                    </a>
                  </td>
                </tr>)}
              </tbody>
            )
          })}
        </table>)}
    </section>
  )
};

LiveBriefs.propTypes = {
  live: React.PropTypes.array.isRequired
};

const formatDate = date => {
  let dateObj = parse(new Date(date || ''));
  return format(dateObj, 'dddd D MMMM YYYY')
};
