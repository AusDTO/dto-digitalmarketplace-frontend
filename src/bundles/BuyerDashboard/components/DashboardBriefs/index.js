import {DashboardBriefs} from './DashboardBriefs';

export default DashboardBriefs;
