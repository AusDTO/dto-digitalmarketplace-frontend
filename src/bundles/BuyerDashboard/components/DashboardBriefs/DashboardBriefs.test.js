import React from 'react';
import renderer from 'react-test-renderer';

import sampleState from './DashboardBriefs.json'
import {LiveBriefs, ClosedBriefs, DraftBriefs} from './DashboardBriefs';


test('LiveBriefs renders with props', () => {
  const state = Object.assign(sampleState, {})
   const component = renderer.create(
     <LiveBriefs live={state.team.briefs.live} />
   )
   let tree = component.toJSON()
   expect(tree).toMatchSnapshot()
 })

test('LiveBriefs renders with no props', () => {
  const liveBriefs = []
   const component = renderer.create(
     <LiveBriefs live={liveBriefs} />
   )
   let tree = component.toJSON()
   expect(tree).toMatchSnapshot()
 })

 test('ClosedBriefs renders with props', () => {
  const state = Object.assign(sampleState, {})
   const component = renderer.create(
     <ClosedBriefs closed={state.team.briefs.closed} />
   )
   let tree = component.toJSON()
   expect(tree).toMatchSnapshot()
 })

test('ClosedBriefs renders with no props', () => {
  const closedBriefs = []
   const component = renderer.create(
     <ClosedBriefs closed={closedBriefs} />
   )
   let tree = component.toJSON()
   expect(tree).toMatchSnapshot()
 })

 test('DraftBriefs renders with props', () => {
  const state = Object.assign(sampleState, {})
   const component = renderer.create(
     <DraftBriefs draft={state.team.briefs.draft} />
   )
   let tree = component.toJSON()
   expect(tree).toMatchSnapshot()
 })

test('DraftBriefs renders with no props', () => {
  const draftBriefs = []
   const component = renderer.create(
     <DraftBriefs draft={draftBriefs} />
   )
   let tree = component.toJSON()
   expect(tree).toMatchSnapshot()
 })




