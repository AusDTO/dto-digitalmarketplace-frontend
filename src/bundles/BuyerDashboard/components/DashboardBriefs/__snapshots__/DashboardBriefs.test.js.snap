// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClosedBriefs renders with no props 1`] = `
<section>
  <h2
    className="au-display-lg"
  >
    Closed briefs
  </h2>
  You don't have any closed briefs.
</section>
`;

exports[`ClosedBriefs renders with props 1`] = `
<section>
  <h2
    className="au-display-lg"
  >
    Closed briefs
  </h2>
  <table
    styleName="brief-table summary-item-body"
  >
    <thead
      styleName="summary-item-field-heading"
    >
      <tr>
        <th
          styleName="left-header"
        >
          Brief Overview
        </th>
        <th
          styleName="center-header"
        >
          Closed
        </th>
        <th
          styleName="right-header"
        >
          Work Order
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="summary-item-row"
      >
        <td
          styleName="item-field-first-full"
        >
          <h5
            className="au-display-xs"
          >
            Brief preview
          </h5>
          <a
            href="/buyers/frameworks/digital-service-professionals/requirements/digital-outcome/119"
          >
            Matts Brief
          </a>
        </td>
        <td
          styleName="item-field-second-left"
        >
          <h5
            className="au-display-xs"
          >
            Closed
          </h5>
          Invalid Date
        </td>
        <td
          styleName="item-field-third-right"
        >
          <h5
            className="au-display-xs"
          >
            Work order
          </h5>
          <span>
            <a
              href="/buyers/frameworks/digital-service-professionals/requirements/digital-outcome/119/responses"
            >
              View responses 
            </a>
          </span>
          <span>
            <a
              href="/buyers/frameworks/digital-service-professionals/requirements/digital-outcome/119/work-orders/create"
            >
              Create work order
            </a>
          </span>
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;

exports[`DraftBriefs renders with no props 1`] = `
<section>
  <h2
    className="au-display-lg"
  >
    Drafts
  </h2>
  You haven't started any opportunities.
</section>
`;

exports[`DraftBriefs renders with props 1`] = `
<section>
  <h2
    className="au-display-lg"
  >
    Drafts
  </h2>
  <table
    styleName="brief-table summary-item-body"
  >
    <thead
      styleName="summary-item-field-heading"
    >
      <tr>
        <th
          styleName="left-header"
        >
          Brief Overview
        </th>
        <th
          styleName="center-header"
        >
          Created
        </th>
        <th
          styleName="right-header"
        >
          Questions left
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="summary-item-row"
      >
        <td
          styleName="item-field-first-full"
        >
          <h5
            className="au-display-xs"
          >
            Brief overview
          </h5>
          <a
            href="/buyers/frameworks/digital-service-professionals/requirements/digital-outcome/119"
          >
            Matts Brief
          </a>
        </td>
        <td
          styleName="item-field-second-left"
        >
          <h5
            className="au-display-xs"
          >
            Created
          </h5>
          Thursday 6 October 2016
        </td>
        <td
          styleName="item-field-third-right"
        >
          <h5
            className="au-display-xs"
          >
            Questions left
          </h5>
          <div>
            17
             required
          </div>
          <div>
            7
             optional
          </div>
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;

exports[`LiveBriefs renders with no props 1`] = `
<section
  styleName="live-brief-container"
>
  <h2
    className="au-display-lg"
  >
    Live briefs
  </h2>
  You don't have any live briefs.
</section>
`;

exports[`LiveBriefs renders with props 1`] = `
<section
  styleName="live-brief-container"
>
  <h2
    className="au-display-lg"
  >
    Live briefs
  </h2>
  <table
    styleName="brief-table summary-item-body"
  >
    <thead
      styleName="summary-item-field-heading"
    >
      <tr>
        <th
          styleName="left-header"
        >
          Brief Overview
        </th>
        <th
          styleName="center-header"
        >
          Closing
        </th>
        <th
          styleName="right-header"
        >
          Q&A
        </th>
      </tr>
    </thead>
    <tbody>
      <tr
        className="summary-item-row"
      >
        <td
          styleName="item-field-first-full"
        >
          <h5
            className="au-display-xs"
          >
            Brief preview
          </h5>
          <a
            href="/2/brief/119/published"
          >
            Matts Brief
          </a>
        </td>
        <td
          styleName="item-field-second-left"
        >
          <h5
            className="au-display-xs"
          >
            Closing
          </h5>
          Invalid Date
        </td>
        <td
          styleName="item-field-third-right"
        >
          <h5
            className="au-display-xs"
          >
            Q&A
          </h5>
          <a
            href="/buyers/frameworks/digital-marketplace/requirements/digital-outcome/119/supplier-questions"
          >
            Answer a question
          </a>
        </td>
      </tr>
    </tbody>
  </table>
</section>
`;
